src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -2377,4 +2377,110 @@ def test_containerapp_max_inactive_revisions(self, resource_group):
         self.cmd(f'containerapp update -g {resource_group} -n {app} --cpu 0.25 --memory 0.5Gi')
         self.cmd(f'containerapp show -g {resource_group} -n {app}', checks=[JMESPathCheck("properties.configuration.maxInactiveRevisions", maxInactiveRevisions)])
 
-        
\ No newline at end of file
+
+class ContainerappRuntimeTests(ScenarioTest):
+    def __init__(self, *arg, **kwargs):
+        super().__init__(*arg, random_config_dir=True, **kwargs)
+
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="northcentralus")
+    def test_containerapp_runtime_java_metrics_create(self, resource_group):
+        TEST_LOCATION = "northcentralusstage"
```

Comment:
The default value of TEST_LOCATION is already `northcentralusstage`, no need to set again.